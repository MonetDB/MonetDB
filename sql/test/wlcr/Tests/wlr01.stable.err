stderr of test 'wlr01` in directory 'sql/test/wlcr` itself:


# 21:37:58 >  
# 21:37:58 >  "/usr/bin/python2" "wlr01.py" "wlr01"
# 21:37:58 >  

# builtin opt 	gdk_dbpath = /export/scratch1/mk/wlcr//Linux/var/monetdb5/dbfarm/demo
# builtin opt 	gdk_debug = 0
# builtin opt 	gdk_vmtrim = no
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 47157
# cmdline opt 	mapi_usock = /var/tmp/mtest-16939/.s.monetdb.47157
# cmdline opt 	gdk_dbpath = /export/scratch1/mk/branches/Nov2019//Linux/var/MonetDB/mTests_sql_test_wlcr-clone

# 21:37:59 >  
# 21:37:59 >  "Done."
# 21:37:59 >  

