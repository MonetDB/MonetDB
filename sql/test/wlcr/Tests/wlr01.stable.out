stdout of test 'wlr01` in directory 'sql/test/wlcr` itself:


# 21:37:58 >  
# 21:37:58 >  "/usr/bin/python2" "wlr01.py" "wlr01"
# 21:37:58 >  

# MonetDB 5 server v11.26.0
# This is an unreleased version
# Serving database 'mTests_sql_test_wlcr-clone', using 8 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 128bit integers
# Found 15.589 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2017 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://vienna.da.cwi.nl:36435/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-32399/.s.monetdb.36435
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded
# SQL catalog created, loading sql scripts once
# loading sql script: 09_like.sql
# loading sql script: 10_math.sql
# loading sql script: 11_times.sql
# loading sql script: 12_url.sql
# loading sql script: 13_date.sql
# loading sql script: 14_inet.sql
# loading sql script: 15_querylog.sql
# loading sql script: 16_tracelog.sql
# loading sql script: 17_temporal.sql
# loading sql script: 18_index.sql
# loading sql script: 20_vacuum.sql
# loading sql script: 21_dependency_functions.sql
# loading sql script: 22_clients.sql
# loading sql script: 23_skyserver.sql
# loading sql script: 25_debug.sql
# loading sql script: 26_sysmon.sql
# loading sql script: 27_rejects.sql
# loading sql script: 39_analytics.sql
# loading sql script: 39_analytics_hge.sql
# loading sql script: 40_geom.sql
# loading sql script: 40_json.sql
# loading sql script: 40_json_hge.sql
# loading sql script: 41_md5sum.sql
# loading sql script: 45_uuid.sql
# loading sql script: 46_gsl.sql
# loading sql script: 46_profiler.sql
# loading sql script: 51_sys_schema_extension.sql
# loading sql script: 60_wlcr.sql
# loading sql script: 72_fits.sql
# loading sql script: 74_netcdf.sql
# loading sql script: 75_lidar.sql
# loading sql script: 75_shp.sql
# loading sql script: 75_storagemodel.sql
# loading sql script: 80_statistics.sql
# loading sql script: 80_udf.sql
# loading sql script: 80_udf_hge.sql
# loading sql script: 85_bam.sql
# loading sql script: 90_generator.sql
# loading sql script: 90_generator_hge.sql
# loading sql script: 99_system.sql
# MonetDB/SQL module loaded
#select 'hello';
% .%2 # table_name
% %2 # name
% char # type
% 5 # length
[ "hello"	]
#/export/scratch1/mk/branches/Nov2019//Linux/var/MonetDB/mTests_sql_test_wlcr/wlc.config
#logs=/export/scratch1/mk/branches/Nov2019//Linux/var/MonetDB/mTests_sql_test_wlcr/wlc_logs/
#id=5
#write=2019-09-14T22:22:23.000
#state=1
#batches=3
#beat=0
#
#/export/scratch1/mk/branches/Nov2019//Linux/var/MonetDB/mTests_sql_test_wlcr/wlc_logs/mTests_sql_test_wlcr_000000000000
#    wlr.transaction(0:lng, "2019-09-14T22:22:23.000":str, "monetdb":str);
#    wlr.catalog("create table tmp0(i int, s string);":str);
#    wlr.commit();
#
#/export/scratch1/mk/branches/Nov2019//Linux/var/MonetDB/mTests_sql_test_wlcr/wlc_logs/mTests_sql_test_wlcr_000000000001
#    wlr.transaction(1:lng, "2019-09-14T22:22:23.000":str, "monetdb":str);
#    wlr.action("insert into tmp0 values(1,\\'gaap\\'), (2,\\'sleep\\');":str);
#    wlr.append("sys":str, "tmp0":str, "i":str, 1:int, 2:int);
#    wlr.append("sys":str, "tmp0":str, "s":str, "gaap":str, "sleep":str);
#    wlr.commit();
#
#/export/scratch1/mk/branches/Nov2019//Linux/var/MonetDB/mTests_sql_test_wlcr/wlc_logs/mTests_sql_test_wlcr_000000000002
#    wlr.transaction(2:lng, "2019-09-14T22:22:23.000":str, "monetdb":str);
#    wlr.catalog("drop table tmp0;":str);
#    wlr.commit();
#    wlr.transaction(3:lng, "2019-09-14T22:22:23.000":str, "monetdb":str);
#    wlr.catalog("create table tmp(i int, s string);":str);
#    wlr.commit();
#    wlr.transaction(4:lng, "2019-09-14T22:22:23.000":str, "monetdb":str);
#    wlr.action("insert into tmp values(1,\\'hello\\'), (2,\\'world\\');":str);
#    wlr.append("sys":str, "tmp":str, "i":str, 1:int, 2:int);
#    wlr.append("sys":str, "tmp":str, "s":str, "hello":str, "world":str);
#    wlr.commit();
#

# 21:37:59 >  
# 21:37:59 >  "Done."
# 21:37:59 >  

